src/status_im2/contexts/wallet/account/view.cljs

```diff
@@ -58,7 +58,7 @@
            :customization-color :blue}]
          [quo/wallet-graph {:time-frame :empty}]
          [quo/wallet-ctas
-          {:send-action   #(rf/dispatch [:open-modal :wallet-select-address])
+          {:send-action   #(rf/dispatch [:open-modal :wallet-select-address (:address account)])
```

Comment:
do we need to pass `address` now. 
@smohamedjavid merged pr: https://github.com/status-im/status-mobile/pull/17798/files#diff-a1b0c995469b4f0ced6921933bce6e8d2cd860beb7871a6367ab9e8bb62a07cd

with sub `(rf/sub [:wallet/current-viewing-account])`